apps/site/assets/ts/stop/__tests__/components/DepartureTimesTest.tsx

```diff
@@ -339,299 +336,6 @@ describe("DepartureTimes", () => {
     });
   });
 
-  describe("infoToDisplayTime", () => {
```

Comment:
If only the formatting is changing I think some of these tests are still valid

Reply:
@i0sea Did you finish reading the PR? This function is deleted.